src/producto.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*- 
+#----------------------------------------------------------------------------
+# Este archivo producto.py contiene la implementación de la clase Producto.
+#----------------------------------------------------------------------------
+# @author: Silvio Orozco @sorozcov 
+# @created_date: 20/11/2021
+# @version: '1.0'
+# ---------------------------------------------------------------------------
+import time
+
+
+class Producto:
+    """
+    Clase utilizada para representar un Producto de Supermercado.
+    ...
+
+    Atributos
+    ----------
+   
+    Publicos
+
+    id: str
+        uuid o id que identifica cada producto como único
+    nombre : str
+        el nombre común del producto
+    precio : float
+        el precio del producto
+    cantidad : int
+        cantidad del producto que se encuentra disponible
+    cantidad_bajo_inventario: int
+        cantidad mínima de inventario para generar un aviso de bajo inventario
+    cantidad_alto_inventario: int
+        cantidad máxima de inventario para generar un aviso de alto inventario
+    cantidad_ultimo_pedido: int
+        cantidad solicitada en el ultimo pedido
+    
+    Privadas
+
+    total_ventas: int
+        cantidad total vendida del producto desde cierta fecha
+    fecha_total_ventas: date
+        fecha desde ultima actualizacion de total ventas a 0
+
+
+    Métodos
+
+    venta:
+        Método que elimina una cantidad de producto al ser vendido y añade al total de ventas. Revisa si es necesario generar un aviso.
```

Comment:
¿Dónde habla de avisos en #9 y #19? ¿Podéis limitaros a hacer lo que piden, y realmente *discutir* qué es lo necesario para la lógica de negocio en los issues correspondientes?

Reply:
Se ha revisado de nuevo, realmente estaba intentando avanzar en #13 y #14, mezclando desordenamente conceptos.